src/guides/v2.3/rest/tutorials/bundle-product/create-bundle-products.md

```diff
@@ -0,0 +1,316 @@
+                      "sku": "24-MB01",
+                      "option_id": 21,
+                      "qty": 1,
+                      "position": 1,
+                      "is_default": true,
+                      "price": 34,
+                      "price_type": null,
+                      "can_change_quantity": 1
+                   },
+                   {
+                      "id": "42",
+                      "sku": "24-MB04",
+                      "option_id": 21,
+                      "qty": 1,
+                      "position": 2,
+                      "is_default": false,
+                      "price": 32,
+                      "price_type": null,
+                      "can_change_quantity": 1
+                   }
+                ]
+             }
+          ]
+       },
+       "options": [],
+       "media_gallery_entries": [],
+       "custom_attributes": [
+          {
+             "attribute_code": "shipment_type",
+             "value": "0"
+          },
+          {
+             "attribute_code": "options_container",
+             "value": "container2"
+          },
+          {
+             "attribute_code": "price_view",
+             "value": "34"
+          },
+          {
+             "attribute_code": "url_key",
+             "value": "new-bundle-product"
+          },
+          {
+             "attribute_code": "msrp_display_actual_price_type",
+             "value": "0"
+          },
+          {
+             "attribute_code": "required_options",
+             "value": "0"
+          },
+          {
+             "attribute_code": "has_options",
+             "value": "0"
+          },
+          {
+             "attribute_code": "meta_title",
+             "value": "New bundle"
+          },
+          {
+             "attribute_code": "sku_type",
+             "value": "0"
+          },
+          {
+             "attribute_code": "meta_keyword",
+             "value": "New bundle"
+          },
+          {
+             "attribute_code": "price_type",
+             "value": "0"
+          },
+          {
+             "attribute_code": "meta_description",
+             "value": "New bundle"
+          },
+          {
+             "attribute_code": "tax_class_id",
+             "value": "2"
+          },
+          {
+             "attribute_code": "weight_type",
+             "value": "0"
+          },
+          {
+             "attribute_code": "category_ids",
+             "value": [
+                "3",
+                "5"
+             ]
+          },
+          {
+             "attribute_code": "short_description",
+             "value": "<p>New bundle</p>"
+          }
+       ]
+    }
+```
+
+## 2. Upload product image for bundle product
+
+Now that we have created a bundle product, we need to upload product image to it.
+
+1. Update the product image by `SKU`.
+1. The product image should be `base64encode`.
+1. Image supported `type` must be `image/png`.
```

Comment:
What abut image/jpeg?

Reply:
sorry I missed that I will again update & push the changes

Reply:
I have updated & pushed the changes @BarnyShergold